keanu-project/src/main/java/io/improbable/keanu/templating/SequenceBuilder.java

```diff
@@ -277,10 +283,11 @@ private FromCountFactories(Collection<Consumer<SequenceItem>> factories, ItemCou
 
 
         public Sequence build() throws SequenceConstructionException {
-            Sequence sequence = new Sequence(count.getCount());
+            int uniqueSequenceIdentifier = this.hashCode();
```

Comment:
so now that I'm going to use the factories to get the hashcode, it will be different in both the places. I.e. one uses a `Collection` of `Consumers` and the other uses a `Collection` of `BiConsumers`. So I don't think it makes as much sense any more to do this. Do you disagree?

Reply:
Sounds reasonable.